calc_pi.ipynb

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
<p><em>I actually prefer a seed of 47.</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ojustino/dat_pyinthesky/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>